scripts/pod_lib_lint.rb

```diff
@@ -68,6 +68,7 @@ def main(args)
   podspec_file = pod_args[0]
   # Figure out which dependencies are local
   deps = find_local_deps(podspec_file, ignore_local_podspecs.to_set)
+  puts 'podspec_file: ' + podspec_file + '\n podspec_file_deps: ' + deps.join(', ')
```

Comment:
Don't commit these here

Reply:
Reverted in [b704b09](https://github.com/firebase/firebase-ios-sdk/pull/9482/commits/b704b098b6c556806b8d6366265fba774105c885)